k8s/namespaces/wa/wa-task-batch-configuration/demo-00.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-configuration
+spec:
+  values:
+    job:
+      schedule: "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56 * * * *"
```

Comment:
Not sure we want this one to be more frequent?  The Configuration job is more for R1 when it failed.  I think we only care about R2 in `demo`. 

Shouldn't do any harm though.

Reply:
done